src/AmplifyGeofenceControl/index.ts

```diff
@@ -175,6 +176,34 @@ export class AmplifyGeofenceControl {
     this._editingGeofenceId = geofence.id;
   }
 
+  deleteGeofence(id: string): void {
+    // FIXME: delete geofence api call here
+    const listItem = document.getElementById(`list-item-${id}`);
+    removeElement(listItem);
+
+    delete this._loadedGeofences[id];
+
+    this._displayedGeofences = this._displayedGeofences.filter(
+      (geofence) => geofence.id !== id
+    );
+
+    this._updateDisplayedGeofences();
+  }
+
+  deleteSelectedGeofences(): void {
```

Comment:
Where is the call site for this method? From the other variables naming in this control, does it make sense for this to be called `deleteDisplayedGeofences`?

Reply:
Yeah I think that would make sense. Currently there is no call site for it since I don't know what the UX is, waiting on our designer to get back to us

Reply:
Added unit tests in a separate PR since the change was getting kind of big, hope that's okay - https://github.com/aws-amplify/maplibre-gl-js-amplify/pull/75